redisinsight/ui/src/pages/browser/components/key-details/KeyDetailsWrapper.tsx

```diff
@@ -17,17 +17,26 @@ import { refreshListElementsAction } from 'uiSrc/slices/list'
 import KeyDetails from './KeyDetails/KeyDetails'
 
 export interface Props {
-  onCloseKey: () => void;
-  onEditKey: (key: string, newKey: string) => void;
-  onDeleteKey: () => void;
-  keyProp: string | null;
+  isFullScreen: boolean
+  onToggleFullScreen: () => void
+  onCloseKey: () => void
+  onEditKey: (key: string, newKey: string) => void
+  onDeleteKey: () => void
+  keyProp: string | null
 }
 
-const KeyDetailsWrapper = ({ onCloseKey, onEditKey, onDeleteKey, keyProp }: Props) => {
+const KeyDetailsWrapper = (
+  {
```

Comment:
Destructuring from *props* variable pls, not inside round brackets

Reply:
done